compile.ts

```diff
@@ -0,0 +1,26 @@
+const specifier = Deno.args[0];
+const out = Deno.args[1];
+const [_, bundle] = await Deno.bundle(specifier);
+
+const originalBinaryPath = Deno.execPath();
+const originalBin = await Deno.readFile(originalBinaryPath);
+
+const encoder = new TextEncoder();
```

Comment:
For first pass I suggest to not have bundling done automatically and thus all of this logic should be moved to Rust

Reply:
Yeah. Did that now. This was just 3AM POC :upside_down_face: